include/date/tz.h

```diff
@@ -49,7 +49,7 @@
 
 #ifndef HAS_REMOTE_API
 #  if USE_OS_TZDB == 0
-#    ifdef _WIN32
+#    if defined _WIN32 || defined __ANDROID__
```

Comment:
I also have somewhat mixed feelings about this #ifdef in general (not your addition of Android):

* On one side, Android has no curl built-in, unlike MacOS,
* On the other, `HAS_REMOTE_API` is also a CMake option, and these kind of defaults only mask the explicit configuration intent

@HowardHinnant WDYT?

Reply:
I'm ok with this.